kolibri/core/errorreports/tasks.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+
+import requests
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import connection
+from requests.exceptions import ConnectionError
+from requests.exceptions import RequestException
+from requests.exceptions import Timeout
+
+from .models import ErrorReports
+from kolibri.core.tasks.decorators import register_task
+from kolibri.core.utils.urls import join_url
+
+logger = logging.getLogger(__name__)
+
+
+def serialize_error_reports_to_json_response(errors):
+    errors_list = []
+    for error in errors:
+        errors_list.append(
+            {
+                "error_from": error.error_from,
+                "error_message": error.error_message,
+                "traceback": error.traceback,
+                "first_occurred": error.first_occurred,
+                "last_occurred": error.last_occurred,
+                "no_of_errors": error.no_of_errors,
+            }
+        )
+    return json.dumps(errors_list, cls=DjangoJSONEncoder)
+
+
+def mark_errors_as_sent(errors):
```

Comment:
Theres not much benefit in having this as a method of its own. We can perform the queryset update within the `ping_error_reports` task. Richard previous asked about this!